bin/cython_freeze

```diff
@@ -45,25 +46,25 @@ print """\
 #else
 # define MODINIT(name)  PyInit_ ## name
 #endif
-"""
+""")
 
 for name in modules:
-    print "PyMODINIT_FUNC MODINIT(%s) (void);" % name
+    print("PyMODINIT_FUNC MODINIT(%s) (void);" % name)
 
-print """
-static struct _inittab inittab[] = {"""
+print("""
+static struct _inittab inittab[] = {""")
 
 for name in modules:
-    print '    {"%(name)s", MODINIT(%(name)s)},' % {'name' : name}
+    print('    {"%(name)s", MODINIT(%(name)s)},' % {'name' : name})
 
-print """    {NULL, NULL}
+print("""    {NULL, NULL}
 };
-""",
+""", end=' ')
```

Comment:
Why a space character and not just the empty string? (I guess that applies in all cases below.)